forest/Cargo.toml

```diff
@@ -64,6 +64,7 @@ once_cell              = "1.14"
 pbr                    = "1.0.4"
 pretty_env_logger      = "0.4"
 prometheus             = { version = "0.13", features = ["process"] }
+question               = "0.2.2"
```

Comment:
Sorry for not spotting it earlier. This seems to be a pretty niche crate with not many downloads that hasn't been active since 2018. In general, we should rely on more established libraries. Was there a particular use-case that was missing from others? Otherwise we could use e.g. [dialoguer](https://crates.io/crates/dialoguer) which has almost 100x more downloads and the usage seems pretty similar. What do you think?

Reply:
I like dialoguer at a quick glance. The color prompts can also help with UX and seems much better than what I was using before. Going to swap it out

Reply:
they are nice and shiny!